tool/tsh/tsh.go

```diff
@@ -1259,9 +1264,16 @@ func newTraceProvider(cf *CLIConf, command string, ignored []string) (*tracing.P
 
 // onVersion prints version info.
 func onVersion(cf *CLIConf) error {
-	proxyVersion, err := fetchProxyVersion(cf)
-	if err != nil {
-		fmt.Fprintf(cf.Stderr(), "Failed to fetch proxy version: %s\n", err)
+	proxyVersion := ""
+	proxyPublicAddr := ""
+	// Do not check proxy version if client only version parameter is given
```

Comment:
nit: the comment here is actually describing the inverse of the code.

The code says "if we're not in client only mode" and the commend describes what to do if we _are_ in client only mode.

Reply:
Updated